eslint/react/.eslintrc

```diff
@@ -18,13 +18,64 @@
 	},
 	"rules": {
 		"class-methods-use-this": "off",
+		"jsx-quotes": "error",
+		"react/boolean-prop-naming": "error",
+		"react/button-has-type": "error",
+		"react/default-props-match-prop-types": "error",
+		"react/jsx-boolean-value": ["error", "never"],
+		"react/jsx-child-element-spacing": "error",
+		"react/jsx-closing-bracket-location": "error",
+		"react/jsx-closing-tag-location": "error",
+		"react/jsx-curly-brace-presence": "error",
+		"react/jsx-curly-spacing": ["error",  {
+			"when": "always",
+			"children": true
+		}],
+		"react/jsx-equals-spacing": ["error", "never"],
+		"react/jsx-filename-extension": "error",
+		"react/jsx-first-prop-new-line": "error",
+		"react/jsx-fragments": ["error", "element"],
+		"react/jsx-handler-names": "error",
+		"react/jsx-no-bind": "error",
+		"react/jsx-props-no-multi-spaces": "error",
+		"react/jsx-one-expression-per-line": "error",
+		"react/jsx-pascal-case": "error",
+		"react/jsx-sort-default-props": "error",
```

Comment:
no...

jsx-sort-default-props

which is the value of that? it's only going to provide crazy diffs on github

Reply:
I was not sure about this as well, I think it's nice to have them sorted, but it's not really adding that much value. I don't care removing it

Reply:
I like them sorted as well but up to you, I dont feel that strong about it

Reply:
for me it's for the diffs of github and because i don't like the approach of let's add it, autofix is going to solve that. Eslint for me has to teach you in some way to code properly without having to run the fixing for you, because you're every time more used. This is not the kind of rule that teach you anything

Reply:
It's also to get aligned on a convention. If we don't want to apply any convention sorting the props of a Component and leave it open... it's ok to me.

Reply:
If we want a convention let's use one that is useful. I would go with ordering the required properties as first and then the ones non. When you look to an API you expect to find the mandatory options at the beginning and then the ones you can omit. This helps in not having to read everything in the component if you want to play with it. 
```js
PropTypes= {
   ...requiredProps,
   ...nonRequiredOnes
}
```

Reply:
> If we want a convention let's use one that is useful. I would go with ordering the required properties as first and then the ones non. When you look to an API you expect to find the mandatory options at the beginning and then the ones you can omit. This helps in not having to read everything in the component if you want to play with it.
> 
> ```js
> PropTypes= {
>    ...requiredProps,
>    ...nonRequiredOnes
> }
> ```

That's cool! I like it.